Hackitect/questioner
# #163218841 Creates user sign up endpoint

**What does this PR do?**
Creates the user sign up route

**Description of Task to be completed?**
Create and test the REST API `POST /auth/signup` 
The data should be persisted in PostgreSQL

**How should this be manually tested?**
*Clone this repo by running `git clone https://github.com/Hackitect/questioner.git`
*Activate virtual environment and install requirements
*Run the flask app
*Run `pytest` you should get 8pass and 1 fail
*Test the api using postman

**Any background context you want to provide?**
TDD

**What are the relevant pivotal tracker stories?**
https://www.pivotaltracker.com/n/projects/2235430/stories/163218841

**Branch Name**
ft-user-sign-up-163218841
